06-building-a-program-start-to-finish/solving-the-problem/output/README.md

```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: GitHub
+- **Deliver**: a Pull Request showing your implementation of the solution (e.g. InvertedIndex class)
+- **With**: *independently*
+
+#### Link to required Pull Request
+- [Here](https://github.com/andela-aawa/inverted-index/pull/17)
```

Comment:
Okay. Changes can be found [here](https://github.com/andela-aawa/inverted-index/pull/20)